e2e/performTests.ts

```diff
@@ -1,20 +1,22 @@
 import TestCommands from "./TestCommands";
+import { resolve } from "path";
 
 const testFilesExistence = (config: string, files: string[]) => {
   it(`should create proper files from ${config}`, () => {
     expect(files).toMatchSnapshot();
   });
 };
 
-const testFilesContent = (commands: TestCommands, config: string, files: string[]) =>
+const testFilesContent = (commands: TestCommands, config: string, files: string[], rootPath: string) =>
   files.forEach((f) => {
     it(`should create proper ${f} from ${config}`, () => {
       const content = commands.getFileContent(`${commands.relativeRoot}/${f}`);
       const cleaned = content
         .replace(/FABLO_BUILD=(.*?)(\n|$)/g, "FABLO_BUILD=<date with git hash>\n")
         .replace(/FABLO_CONFIG=(.*?)(\n|$)/g, "FABLO_CONFIG=<absolute path>\n")
         .replace(/CHAINCODES_BASE_DIR=(.*?)(\n|$)/g, "CHAINCODES_BASE_DIR=<absolute path>\n")
-        .replace(/COMPOSE_PROJECT_NAME=(.*?)(\n|$)/g, "COMPOSE_PROJECT_NAME=<name with timestamp>\n");
+        .replace(/COMPOSE_PROJECT_NAME=(.*?)(\n|$)/g, "COMPOSE_PROJECT_NAME=<name with timestamp>\n")
+        .replace(new RegExp(rootPath, "g"), "<absolute path>");
```

Comment:
sprytnie z tym rootPath. Zadziała to może też na `FABLO_CONFIG` i `CHAINCODES_BASE_DIR`? bo jeśli tak, to bym tamte replacy proponował wywalić

druga rzecz - schowałbym ten rootPath tutaj, tzn nie jako parameter, tylko tutaj resolvował `const rootPath = resolve(__dirname + "/../");`

Reply:
To nie zadziałało, tam były jakieś inne wartości.